public/scss/_forms.scss

```diff
@@ -144,16 +144,18 @@ label:focus ~ .input-help-focused {
   opacity: 1;
 }
 
-.card-icon {
-    opacity: 0;
-    position: absolute;
-    right: 8px;
-    top: 50%;
-    margin-top: -13px;
-    z-index: 10;
-}
+form {
```

Comment:
Thats only the position of the card icon within its parent I believe? In the case of the email it shoves it into the bottom right hand corner miles away from where it should be.